src/modelmanager.cpp

```diff
@@ -479,31 +479,20 @@ Status ModelManager::validateUserSettingsInSingleModelCliGraphStart(const Models
 }
 
 Status ModelManager::processMediapipeConfig(const MediapipeGraphConfig& config, std::set<std::string>& mediapipesInConfigFile, MediapipeFactory& factory) {
```

Comment:
We probably dont need `mediapipesInConfigFile` if we dont use it in this function anymore